web/docs/deployment/deployment-methods/wasp-deploy/ci-cd.md

```diff
@@ -0,0 +1,115 @@
+---
+title: CI/CD Scenarios
+---
+
+import { WaspDeployProvidersGrid } from './WaspDeployProvidersGrid';
+
+Once you deploy your application with `wasp deploy <provider> launch` locally, you can use CI/CD platforms like Github Actions to re-deploy your application automatically whenever changes are pushed to your repository.
+
+## Re-deploying from CI/CD
+
+### Prerequisites
+
+Make sure you set up your application locally before you can deploy it using CI/CD. You do this by running `wasp deploy <provider> launch` which creates your application in the deployment provider and deploys it.
+
+<WaspDeployProvidersGrid />
+
+### Deployment steps
+
+To automate deployment, you need to create a workflow file in your repository that specifies the deployment process when a new commit is pushed to the repository. 
+
+The workflow needs to include the following steps:
+
+1. Checkout the code from the repository
+2. Install Node.js and the Wasp CLI
+3. Install any provider-specific dependencies
+4. Deploy the application using `wasp deploy <provider> deploy`
+
+You need to make sure that the **provider-specific environment variables** (for example, provider API keys) are set up correctly before deploying. We show which variables are required and how to set them up in the Github Actions [example](#github-actions-workflow).
```

Comment:
On the first pass, this paragraph threw me off. I was confused on what the provider-specific environment variables are, since we have Wasp enviroment variables (e.g. `JWT_SECRET`, `WASP_FRONTEND_URL`), App environment variables (e.g. `OPENAI_API_TOKEN`), and now Provider-specific environment variables (which now I understand to be `FLY_API_TOKEN` etc). It all mixes up in my head.

Linking to the example immediately after also was a bit weird because I was already readying my mind for a context change, preparing to read a whole new document about env vars that explained this.

I don't have a specific suggestion for you, but maybe we can try rewriting this paragraph and moving it around so this doesn't happen.